CMakeLists.txt

```diff
@@ -293,6 +277,58 @@ ELSE ()
 	ENDIF()
 ENDIF()
 
+IF(USE_TLS STREQUAL "ON")
+	IF (SECURITY_FOUND)
+		# OS X 10.7 and older do not have some functions we use, fall back to OpenSSL there
+		CHECK_LIBRARY_EXISTS("${SECURITY_DIRS}" SSLCreateContext "Security/SecureTransport.h" HAVE_NEWER_SECURITY)
```

Comment:
Aforementioned issue. Forcing the `SecureTransport` backend will skip this test.